src/main/java/com/ipiecoles/java/java220/Commercial.java

```diff
@@ -0,0 +1,93 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+public class Commercial extends Employe {
+    private Double caAnnuel;
+    private Integer performance;
+
+    public Commercial(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Double caAnnuel) {
+        super(nom, prenom, matricule, dateEmbauche, salaire);
+        this.caAnnuel = caAnnuel;
+    }
+
+    public Commercial(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Double caAnnuel, Integer perf) {
+        super(nom, prenom, matricule, dateEmbauche, salaire);
+        this.caAnnuel = caAnnuel;
+        this.performance = perf;
+    }
+
+    public Double getCaAnnuel() {
+        return caAnnuel;
+    }
+
+    public void setCaAnnuel(Double caAnnuel) {
+        this.caAnnuel = caAnnuel;
+    }
+
+    public Integer getPerformance() {
+        return performance;
+    }
+
+    public void setPerformance(Integer performance) {
+        this.performance = performance;
+    }
+
+    @Override
+    public Double getPrimeAnnuelle() {
+        if(this.getCaAnnuel() == null)
+            return 500d;
+        Double resultat = this.getCaAnnuel() * 0.05;
+        if(resultat < 500)
+            return 500d;
+        else
+            return Math.ceil(resultat);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        final Commercial c = (Commercial) obj;
```

Comment:
Cette instruction peut planter si obj n'est pas un commercial. Utilise instanceof avant pour t'en assurer
```suggestion
        if(!obj instanceof Commercial){
            return false;
        }
        final Commercial c = (Commercial) obj;
```